src/app-layer-htp-range.h

```diff
@@ -40,7 +40,7 @@ typedef struct HttpRangeContainerBuffer {
     /** offset of bytes written in buffer (relative to the start of the range) */
     uint64_t offset;
     /** number of gaped bytes */
-    uint64_t gap;
+    uint32_t gap;
```

Comment:
a file can be larger than 4GiB, so a gap in it's data could also exceed 4GiB

Reply:
ok, will saturate it...